src/stores/utils/gas.js

```diff
@@ -0,0 +1,25 @@
+export async function fetchGasPrice({ bridgeContract, oracleFn }) {
+  let gasPrice = null
+  try {
+    gasPrice = await oracleFn()
+  } catch (e) {
+    console.error(`Gas Price API is not available. ${e.message}`)
+
+    try {
+      gasPrice = await bridgeContract.methods.gasPrice().call()
+    } catch (e) {
+      console.error(`There was a problem getting the gas price from the contract. ${e.message}`)
+    }
+  }
+  return gasPrice
+}
+
+export async function fetchGasPriceFromOracle(oracleUrl, speedType) {
+  const response = await fetch(oracleUrl)
+  const json = await response.json()
+  const gasPrice = json[speedType]
+  if (!gasPrice) {
+    throw new Error(`Response from Oracle didn't include gas price for ${speedType} type.`)
+  }
+  return gasPrice
```

Comment:
GasPrice returned from Oracle is on `gwei` while gas price on contract and fallback are on `wei` values. We had the same issue on bridge-nodejs an solved converting returned oracle value to wei https://github.com/poanetwork/bridge-nodejs/pull/85/commits/2173d84694270ca032c34fcb1e6dc4e16fe81201

Then we need to remove the convertion from `gwei` on:
https://github.com/poanetwork/bridge-ui/blob/af91746da3c0df9324a00b44d5bbf61fffd3744f/src/stores/GasPriceStore.js#L58